source/simulator/core.cc

```diff
@@ -1219,6 +1219,15 @@ namespace aspect
     else
       coupling[x.pressure][x.pressure] = DoFTools::always;
 
+    // the system_preconditioner matrix is only used for the Stokes
```

Comment:
would it make more sense to have this comment in line 1195 instead?

Reply:
I put it here because I would have expected the temperature/composition block description to come right here, after the Stokes description. The comment was meant to explain why there is not in fact anything right where the comment is.

But I obviously don't feel strongly. You decide.

Reply:
same here. ;-)